tools/walletextension/storage/storage.go

```diff
@@ -0,0 +1,49 @@
+package storage
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/obscuronet/go-obscuro/go/rpc"
+)
+
+type Storage struct {
```

Comment:
would be good to define an interface for the Storage.
There should be 2 methods:
- SaveUserVK
- GetUserVKs
(basically renaming the Save and Get methods you have already)

Reply:
renamed mehods